collectors/0/iostat.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.6
```

Comment:
of course you're right :) but i had to make it work on Rhel5... that is why i had to force 2.6 otherwise it doesn't run...